<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildingUseMapDialog</class>
 <widget class="QDialog" name="BuildingUseMapDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Building Use Map</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save or discard changes</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="buildUseMapTable">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>20</y>
     <width>530</width>
     <height>240</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="refreshToolButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>20</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Refresh all records</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="deleteToolButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>88</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Reset selected record</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="deleteAllToolButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>224</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Reset all records</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="multiMapToolButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>156</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Multi map selection</string>
   </property>
   <property name="text">
    <string>^</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buildUseMapTable</tabstop>
  <tabstop>refreshToolButton</tabstop>
  <tabstop>deleteToolButton</tabstop>
  <tabstop>multiMapToolButton</tabstop>
  <tabstop>deleteAllToolButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BuildingUseMapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BuildingUseMapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
